src/pyedb/dotnet/edb_core/edb_data/raptor_x_simulation_setup_data.py

```diff
@@ -0,0 +1,448 @@
+
+    @property
+    def eliminate_slit_per_hole(self):
+        """This is a setting that internally simplifies layouts with strain relief or thermal relief slits and
+        holes. It will examine each hole separately against the whole polygon it belongs to.
+        If the area of the hole is below the threshold defined in this setting, then the hole will be filled.
+        Units: unitless.
+        """
+        return self._advanced_settings.EliminateSlitPerHoles
+
+    @eliminate_slit_per_hole.setter
+    def eliminate_slit_per_hole(self, value):
+        self._advanced_settings.EliminateSlitPerHoles = EdbValue(value).tofloat
+
+    @property
+    def mesh_frequency(self):
+        """User can override the default meshing applied by setting a custom frequency for mesh generation.
+        Example: "1GHz".
+        """
+        return self._advanced_settings.MeshFrequency
+
+    @mesh_frequency.setter
+    def mesh_frequency(self, value):
+        self._advanced_settings.MeshFrequency = EdbValue(value).tostring
+
+    @property
+    def net_settings_options(self):
+        """A list of Name, Value pairs that stores advanced option."""
+        return [val for val in list(self._advanced_settings.NetSettingsOptions)]
+
+    @net_settings_options.setter
+    def net_settings_options(self, value):
+        if isinstance(value, list):
+            self._advanced_settings.NetSettingsOptions = convert_py_list_to_net_list(value)
+
+    @property
+    def override_shrink_fac(self):
+        """Set the shrink factor explicitly, that is, review what-if scenarios of migrating to half-node
+        technologies.
+        Units: unitless.
+        """
+        return self._advanced_settings.OverrideShrinkFac
+
+    @override_shrink_fac.setter
+    def override_shrink_fac(self, value):
+        self._advanced_settings.OverrideShrinkFac = EdbValue(value).tofloat
+
+    @property
+    def plane_projection_factor(self):
+        """To eliminate unnecessary mesh complexity of "large" metal planes and improve overall extraction time,
+        user can define the mesh of certain planes using a combination of the Plane Projection Factor and
+        settings of the Nets Advanced Options.
+        Units: unitless.
+        """
+        return self._advanced_settings.PlaneProjectionFactor
+
+    @plane_projection_factor.setter
+    def plane_projection_factor(self, value):
+        self._advanced_settings.PlaneProjectionFactor = EdbValue(value).tofloat
+
+    @property
+    def use_accelerate_via_extraction(self):
+        """Setting this option will simplify/merge neighboring vias before sending the layout for processing
+        to the mesh engine and to the EM engine.
+        """
+        return self._advanced_settings.UseAccelerateViaExtraction
+
+    @use_accelerate_via_extraction.setter
+    def use_accelerate_via_extraction(self, value):
+        if isinstance(value, bool):
+            self._advanced_settings.UseAccelerateViaExtraction = value
```

Comment:
Can you log an error if value is not bool ?

Reply:
Done